src/libraries/System.Private.Xml/src/System/Xml/Xsl/XsltOld/NumberAction.cs

```diff
@@ -509,11 +509,6 @@ private static string Format(ArrayList numberlist, List<FormatInfo?>? tokens, st
             }
             if (groupingSep != null)
             {
-                if (groupingSep.Length > 1)
-                {
-                    // It is a breaking change to throw an exception, SQLBUDT 324367
-                    //throw XsltException.Create(SR.Xslt_CharAttribute, "grouping-separator");
-                }
```

Comment:
So we wanted to throw here but can't?

What's the behavior then if the `length > 1`?